src/main/java/com/aquarium/aquarium_backend/Services/UserSensorService.java

```diff
@@ -26,11 +26,15 @@ public List<UserSensors> getAllUsers() {
     return userSensorRepository.findAll();
   }
 
+  public List<UserSensors> getUserSensorsByAquariumId(Long aquariumId) {
+    return userSensorRepository.findUserSensorsByAquariumId(aquariumId);
+  }
+
   public boolean addUserSensor(Long aquariumId, int sensorTypeId) {
     try {
       var userSensor = new UserSensors();
-      userSensor.setAquarium(aquariumRepository.findById(aquariumId).orElse(null));
-      userSensor.setSensorType(sensorTypeRepository.findById(sensorTypeId).orElse(null));
+      userSensor.setAquarium(aquariumRepository.findById(aquariumId).get());
```

Comment:
Use object initialization with constructor with arguments instead of empty objects with setters

Reply:
Done